test/suite/test_util01.py

```diff
@@ -26,7 +26,7 @@
 # ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
 # OTHER DEALINGS IN THE SOFTWARE.
 
-import string, os, sys, random
+import string, os, sys, random, string
```

Comment:
I assume. you don't mean to (or need to) import the string library twice.

Reply:
Fixed.